### ISSUE TITLE
kill method 

### ISSUE THREAD

**Issue by max107** [malaman/js-image-zoom#8]
https://github.com/malaman/js-image-zoom/blob/master/package/js-image-zoom.js#L171

```
if (zoomLens && zoomDiv) {
    container.removeChild(data.zoomLens.element);
    container.removeChild(data.zoomedImg.element);
}
```

`zoomLens` and `zoomDiv` always `undefined`

**Comment by malaman** [comment]
I see. This is a bug. I will fix it next week and publish updated version.

**Comment by malaman** [comment]
Fixed in 0.4.0 release.
